load_testing/prod_simulator.locustfile.py

```diff
@@ -9,13 +17,13 @@
 # Defaults updated based on measurements from 2021-04-13
 
 RATIOS = {
-    "SIGN_IN": 7263,
-    "SIGN_UP": 813,
-    "SIGN_IN_AND_PROOF": 9,
-    "SIGN_UP_AND_PROOF": 9,
-    "SIGN_IN_USER_NOT_FOUND": 885,
-    "SIGN_IN_INCORRECT_PASSWORD": 885,
-    "SIGN_IN_INCORRECT_SMS_OTP": 79,
+    "SIGN_IN": 10000,
+    "SIGN_UP": 0,
```

Comment:
This makes the mix of load 100% sign_in.   Can we update the defaults?  (Or just drop this entirely and require setting the ratios at run time)

Reply:
Yep, I haven't restored the defaults yet. I'm going through each flow at 100% for a few minutes to make sure things work as expected and will post the results here.